lib/index.d.ts

```diff
@@ -169,20 +174,21 @@ export function encryptMessage(
 ): Promise<EncryptResult<string>>;
 export function encryptMessage(
     options: EncryptOptionsPmcrypto & { armor?: true; detached: true }
-): Promise<EncryptResult<string, undefined, string>>;
+): Promise<EncryptResult<string, undefined, string, string>>;
 export function encryptMessage(
     options: EncryptOptionsPmcrypto & { armor: false; detached?: false }
 ): Promise<EncryptResult<undefined, message.Message, undefined>>;
```

Comment:
This is very nitpicky but I guess you might want to either add or remove one `undefined` here, for consistency?